src/tagging/reducers/index.js

```diff
@@ -0,0 +1,19 @@
+import { combineReducers } from 'redux';
+import { modifySetTags, toggle } from './reducers';
+
+const TAGS = {
```

Comment:
these should move into fixtures files.

Reply:
This shouldn't be part of this component, It was there because of testing purposes. I moved it to stories and demo. Now I pass it as a default state, when creating this component